script/run_bench.py

```diff
@@ -11,8 +11,8 @@
 import sys
 
 # Runs the tests.
-WREN_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
-BENCHMARK_DIR = os.path.join(WREN_DIR, 'benchmark')
+HOME_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Given that "wren" is the directory you'll end up with when you clone the Git repo, I think `WREN_DIR` works pretty well here. I agree with @zeckalpha—I'd intuit `HOME_DIR` to mean `~`.